versions/1/data_stream/fields/fields.spec.yml

```diff
@@ -78,9 +78,13 @@ spec:
         type: string
         examples:
           - '^[a-zA-Z]$'
+      reference:
+        description: External source reference
```

Comment:
Haha, both names are in here 🤦  source and reference.

Reply:
Another option: 

```
external: ecs
```

Reply:
Ok, I renamed to `external`.